.github/workflows/update-styles.yml

```diff
@@ -11,9 +11,42 @@ on:
     - cron: '0 0 * * 3'
 
 jobs:
+  check-token:
+    name: "Check STYLES_WORKFLOW token"
+    runs-on: ubuntu-latest
+    if: ${{ github.repository == 'carpentries/lesson-example' }}
+    outputs:
+      wf: ${{ steps.validate.outputs.wf }}
```

Comment:
I know `wf` is the name you use in the action, so changing it is a bit involved but ultimately, I think it would be best to rename it to something more descriptive, _e.g._ `workflow`.

Reply:
Good point. I can create a redundant output and deprecate the wf output.